api/src/main/resources/underlays/omop_test.yaml

```diff
@@ -0,0 +1,437 @@
+# An initial OMOP underlay schema for development.
+
+name: omop_test
+entities:
```

Comment:
@freemabd would be curious to get your feedback on this

Reply:
This seems logical but very tedious and easy to make a mistake. I assuming this was this done manually? Could this be automated in some way using some sort of Entity object and the BQ OMOP schema files? Would it also make sense to break these into separate yaml files? This file could get pretty large.

Reply:
This was done manually for this v0.
We could write some pretty simple code to take BQ schema and make the dataset/table definitions as well as the simple column attributes for files. What doesn't work was well is things like person.gender, where that's not a direct column on the person table (person.gender_concept_id is). We can develop more tools as we scale this out to make it less manual. There's already checking in place to find errors.
We can split things into multiple yaml files in a later iteration too.